cypress/integration/e2e/article.elements.spec.js

```diff
@@ -48,4 +48,23 @@ describe('Elements', function() {
             getIframeBody().contains('View More on Instagram');
         });
     });
+
+    describe('WEB', function() {
```

Comment:
If we keep the other 'describe WEB as the parent and nest two new ones for, say, describe 'InstagramBlockElement' and 'describe EmbedBlockComponent' then these tests will print [really pretty output](https://jestjs.io/docs/en/api#describename-fn) which makes me feel good inside.

Plus it will help make it clearer what the scope of the tests are.

Maybe like:

```
describe('WEB', function() {
    describe('for InstagramBlockElement', function() {
        // ... etc.
    })
    describe('for EmbedBlockComponent', function() {
        // ... etc.
    })
})
```